cocos/core/splash-screen-webgl.ts

```diff
@@ -125,8 +126,9 @@ export class SplashScreenWebgl {
     }
 
     public main (canvas: HTMLCanvasElement) {
-        if (window._CCSettings && window._CCSettings.splashScreen) {
-            this.setting = window._CCSettings.splashScreen;
+        this.ccSetting = window._CCSettings;
```

Comment:
Please do not save CCSettings, it's too big, only save what you need here. By the way, why you need to save it ?